packages/functional-tests/pages/signin.ts

```diff
@@ -38,12 +38,14 @@ export class SigninPage extends BaseLayout {
     return this.page.getByRole('button', { name: 'Confirm' });
   }
 
+  // Use /Continue with.*Apple/ because of hidden bidi Unicode characters around "Apple" in its accessible name.
```

Comment:
Thank you for the comment.

Reply:
This was a weird one to troubleshoot. Took me a bit to figure out why /Apple/ worked but not "Continue with Apple" when the button was clearly visible in the playwright trace 🤦‍♀️